examples/light-switch-app/light-switch-common/light-switch-app.matter

```diff
@@ -580,6 +580,28 @@ client cluster Groups = 4 {
   readonly attribute bitmap8 nameSupport = 0;
   readonly global attribute attrib_id attributeList[] = 65531;
   readonly global attribute int16u clusterRevision = 65533;
+
+  request struct AddGroupRequest {
```

Comment:
These requests are not used by any commands. Why are they generated?

Reply:
I updated the responses which added the following
command AddGroup(AddGroupRequest): AddGroupResponse = 0;

Were you looking for that?

Reply:
CHIPClusters and CHIPClientCallback no longer generate anything at all for commands on tip.  Please update to tip.....